core/eolearn/core/eoexecution.py

```diff
@@ -205,7 +207,7 @@ def _try_remove_logging(cls, log_path, logger, handler):
                 warnings.warn(f'Failed to end logging with exception: {repr(exception)}', category=EORuntimeWarning)
 
     @classmethod
-    def _execute_workflow(cls, process_args):
+    def _execute_workflow(cls, process_args: object) -> WorkflowResults:
```

Comment:
annotating `process_args` with `object` will make mypy angry whenever you try to do anything specific with it (such as unpacking in line 213) unless you do a type inspection first. I suggest you write out the type of `process_args` in an alias in this file and then annotate accordingly throughout the code.